scripts/signing/sign.sh

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env sh
+set -e
+
+#
+#
+# Upload file to GH Release assets
+#
+#
```

Comment:
I guess this comment can be removed since it doesn't apply here. Maybe can just be Sign assets.